tests/AwardsTest.php

```diff
@@ -32,7 +32,7 @@ public function setUp(): void
     public function testGetAwardsClasses()
     {
         $classes = $this->awardSvc->findAllAwardClasses();
-        $this->assertCount(2, $classes);
+        $this->assertCount(3, $classes);
```

Comment:
https://phpunit.readthedocs.io/en/9.3/assertions.html#assertgreaterthan

Reply:
I found the documentation late last night and was looking to try some of these other methods

Reply:
I went with assertGreaterThanOrEqual(2, $classes) in commit b804a0a, to include the 2 included Classes + any others that are added